portfolio/src/main/webapp/script.js

```diff
@@ -118,35 +119,45 @@ function showModal() {
     let commentForm = document.createElement("form");
     commentForm.setAttribute('method', "post");
     commentForm.setAttribute('action', "/new-comment");
+    let commentAuthor = document.createElement("input");
+    commentAuthor.setAttribute('type', "text");
+    commentAuthor.setAttribute('maxlength', "30");
+    commentAuthor.setAttribute('name', "commentAuthor");
+    commentAuthor.setAttribute('placeholder', "Name");
     let commentText = document.createElement("input");
     commentText.setAttribute('type', "text");
+    commentText.setAttribute('maxlength', "30");
     commentText.setAttribute('name', "commentText");
+    commentText.setAttribute('placeholder', "Comment");
     let submitButton = document.createElement("input");
     submitButton.setAttribute('type', "submit");
     submitButton.setAttribute('value', "Submit");
 
+    commentForm.appendChild(commentAuthor);
+    commentForm.appendChild(document.createElement('br'));
     commentForm.appendChild(commentText);
+    commentForm.appendChild(document.createElement('br'));
     commentForm.appendChild(submitButton);
 
     modalBody.appendChild(commentForm);
   }
 
   // When the user clicks anywhere outside of the modal, close it
-  window.onclick = function(event) {
+  window.onclick = function (event) {
     if (event.target == modal) {
       modal.style.display = "none";
     }
   }
 }
 
-window.onscroll = function() { scrollFunction() };
+window.onscroll = function () { scrollFunction() };
 
 // Expands or contracts the navbar and displays to top button on scroll
 
 function scrollFunction() {
   let scrollToTopButton = document.getElementById("scrollToTopButton");
   let navigationMenu = document.getElementById("navigationMenu");
-  // When the user scrolls down 20px from the top of the document, show the button
+  // When the user scrolls down 310px from the top of the document, show the button
   if (document.body.scrollTop > 310 || document.documentElement.scrollTop > 310) {
```

Comment:
Nit: this `310` should probably be a top-of-function constant instead of an inlined magic number.